Library/ViewModels/LiveVideoViewModelTests.swift

```diff
@@ -10,6 +10,13 @@ private struct TestOTStreamType: OTStreamType {
   fileprivate let streamId: String
 }
 private class TestOTErrorType: NSError, OTErrorType {}
+private struct TestOTSubscriberVideoEventReasonType: OTSubscriberVideoEventReasonType {
```

Comment:
don't think we need to suffix this with `Type` since it's a concrete implementation